api/src/main/java/jakarta/config/ConfigDefault.java

```diff
@@ -0,0 +1,22 @@
+package jakarta.config;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Specify the default value of a configuration member.
+ */
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.METHOD })
+public @interface ConfigDefault {
```

Comment:
What about ConfigValue or ConfigDefaultValue?

Reply:
If it is only available in a `@ConfigMapping` class/interface, I would like move these annotations(`ConfigName`, `ConfigDefault`) into `ConfigMapping` and remove the `Config` prefix, eg. `@Key`, `@DefaultValue`.

Reply:
Use a `Config` prefix everywhere make the codes looks tedious.

Reply:
Sounds reasonable. I'll have a look.

Reply:
https://github.com/jakartaee/config/pull/252